packages/block-library/src/cover/test/edit.js

```diff
@@ -375,4 +378,51 @@ describe( 'Cover block', () => {
 			} );
 		} );
 	} );
+
+	describe( 'isDark settings', () => {
+		test( 'should toggle is-light class if background changed from light to dark', async () => {
+			await setup();
+			const colorPicker = screen.getByRole( 'button', {
+				name: 'Color: White',
+			} );
+			await userEvent.click( colorPicker );
+
+			const coverBlock = screen.getByLabelText( 'Block: Cover' );
+
+			expect( coverBlock ).toHaveClass( 'is-light' );
+
+			await selectBlock( 'Block: Cover' );
+			await userEvent.click(
+				screen.getByRole( 'tab', {
+					name: 'Styles',
+				} )
+			);
+			await userEvent.click( screen.getByText( 'Overlay' ) );
+			const popupColorPicker = screen.getByRole( 'button', {
+				name: 'Color: Black',
+			} );
+			await userEvent.click( popupColorPicker );
+			expect( coverBlock ).not.toHaveClass( 'is-light' );
+		} );
```

Comment:
I don't know why the following is the case as if there is no overlay color it defaults to black, so doesn't seem to make sense to then set it to `is-light`  - but I think we should just preserve the existing behaviour for now for the jitter bug fix.

Reply:
+1 one to fixing that bug separately. We can discuss it and get design input on a dedicated issue for it.